tools/astarte_export/lib/astarte/fetchdata/fetchdata.ex

```diff
@@ -285,9 +285,42 @@ defmodule Astarte.Export.FetchData do
     end
   end
 
+  defp from_native_type(value, :binaryblobarray) when is_list(value) do
+    case value do
+      list when is_list(list) -> Enum.map(list, fn x -> from_native_type(x, :binaryblob) end)
+      _ -> value
+    end
```

Comment:
This check is redundant, the guard in the head of the function makes sure that `is_list(value)` already